app/helpers/content_helper.rb

```diff
@@ -1,6 +1,11 @@
 module ContentHelper
   def path_for_this_page(page)
-    "#{content_url}/#{page.slug}"
+    # Add parent links
```

Comment:
I've added these, but it felt like repeating the same two lines of code that was in the code under test

Reply:
I don't understand the comment.

Generally, the use of a conditional statement should be accommodated with a unit test because it contains logic.

Reply:
The test seems pointless, because all you are really doing is saying 'this is how I think the controller works' and then using the exact same code in the test. Both might be wrong.